pkg/cmd/oauth-server/cmd.go

```diff
@@ -51,24 +55,30 @@ func NewOsinServer(out, errout io.Writer, stopCh <-chan struct{}) *cobra.Command
 		},
 	}
 
+	// Handle Flags
 	flags := cmd.Flags()
-	// This command only supports reading from config
+	options.Audit.AddFlags(flags)
+
 	flags.StringVar(&options.ConfigFile, "config", "", "Location of the osin configuration file to run from.")
 	cmd.MarkFlagFilename("config", "yaml", "yml")
 	cmd.MarkFlagRequired("config")
 
 	return cmd
 }
 
-func (o *OsinServer) Validate() error {
+func (o *OsinServerOptions) Validate() error {
 	if len(o.ConfigFile) == 0 {
 		return errors.New("--config is required for this command")
 	}
 
+	if o.Audit == nil {
```

Comment:
Add validation that makes sure that the audit level does not exceed `Metadata` - we don't want to be logging request and response bodies

Reply:
done